api/v1/contacts.md

```diff
@@ -62,3 +62,117 @@ updatedAtMax | false | String ISO Time | Upper bound (exclusive) of the updatedA
 
 
 ---
+
+
+
+
+### `GET` `/contacts/:id/tags`
+* Returns a list of tags associated to the contact by `:id`.
+
+####  Header
+key|value
+---|---
+`Authorization`| Basic authorization (`Basic ` + base64(`api key` + `:`))
+
+#### Sample Response
+```javascript
+{
+    "status": "success",
+    "requestId": "jLIpDB",
+    "result": [
+        {
+            "id": "1b697008-502f-42bd-836c-ca0813ea4290",
+            "name": "SF",
+            "typeId": "21ea6b07-7fff-452b-ab05-4734505d8eaf",
+            "status": "active"
+        },
+        {
+            "id": "3c83269d-5f37-4607-85af-6cf0c33a284d",
+            "name": "Engineering",
+            "typeId": "56fe0869-c029-4f16-a71f-25fea308ad05",
+            "status": "active"
+        }
+    ]
+}
+```
+
+
+
+---
+
+
+
+
+### `PUT` `/contacts/:id/tags`
+* Returns a list of UserToTag objects after adding a list of tags to the contact by `:id`.
+
+####  Header
+key|value
+---|---
+`Authorization`| Basic authorization (`Basic ` + base64(`api key` + `:`))
+
+#### Sample Request
+```javascript
+[
+    {
+        "id": "1b697008-502f-42bd-836c-ca0813ea4290",
```

Comment:
If this tag is not found what happens?
May be also document the possible error codes it may return.

Reply:
Also added a possible error code response for when tag is not found

Reply:
So if one of the tags are not found what happens?
None of the tags gets added? Only the ones that exists gets added?